application/src/burnham/cli.py

```diff
@@ -148,6 +151,9 @@ def burnham(
                 metrics.test.run.set(test_run)
                 metrics.test.name.set(test_name)
 
+            logger.info("All missions completed. Waiting for telemetry to be sent.")
+            time.sleep(5)
```

Comment:
This currently runs after every mission item in `missions`.

I suggest moving this outside of the for-loop:

```suggestion
        secs = 5
        logger.info("All missions completed.")
        logger.info(f" Waiting {secs}s for telemetry to be sent.")
        time.sleep(secs)
```